diff --git a/node_modules/clipboard-event/index.js b/node_modules/clipboard-event/index.js
index 2cb3c86..09e24ab 100644
--- a/node_modules/clipboard-event/index.js
+++ b/node_modules/clipboard-event/index.js
@@ -1,7 +1,8 @@
-const { EventEmitter } = require('events');
+const {EventEmitter} = require('events');
 const path = require('path');
-const { execFile } = require('child_process');
-
+const {execFile} = require('child_process');
+const lstatSync = require("fs").lstatSync
+const existsSync = require("fs").existsSync
 class ClipboardEventListener extends EventEmitter {
   constructor() {
     super();
@@ -9,23 +10,29 @@ class ClipboardEventListener extends EventEmitter {
   }
 
   startListening() {
-
-    const { platform } = process;
-    if (platform === 'win32') {
-      this.child = execFile(path.join(__dirname, 'platform/clipboard-event-handler-win32.exe'));
+    const {platform} = process;
+    let dir = __dirname
+    console.log(dir)
+    if(existsSync(__dirname.replace(/app\.asar/, "app.asar.unpacked"))) {
+      console.log("prod")
+      dir = __dirname.replace(/app\.asar/, "app.asar.unpacked")
+    }
+    console.log(dir)
+    if(platform === 'win32') {
+      this.child = execFile(path.join(dir, 'platform/clipboard-event-handler-win32.exe'));
     }
-    else if (platform === 'linux') {
-      this.child = execFile(path.join(__dirname, 'platform/clipboard-event-handler-linux'));
+    else if(platform === 'linux') {
+      this.child = execFile(path.join(dir, 'platform/clipboard-event-handler-linux'));
     }
-    else if (platform === 'darwin') {
-      this.child = execFile(path.join(__dirname, 'platform/clipboard-event-handler-mac'));
+    else if(platform === 'darwin') {
+      this.child = execFile(path.join(dir, 'platform/clipboard-event-handler-mac'));
     }
     else {
       throw 'Not yet supported';
     }
 
     this.child.stdout.on('data', (data) => {
-      if (data.trim() === 'CLIPBOARD_CHANGE') {
+      if(data.trim() === 'CLIPBOARD_CHANGE') {
         this.emit('change');
       }
     });
diff --git a/node_modules/clipboard-event/platform/clipboard-event-handler-linux b/node_modules/clipboard-event/platform/clipboard-event-handler-linux
old mode 100644
new mode 100755
